CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+To make contributions to this charm, you'll need a working [development setup](https://juju.is/docs/sdk/dev-setup).
+
+You can create an environment for development with `tox`:
+
+```shell
+tox devenv -e integration
+source venv/bin/activate
+```
+
+## Common Module
+
+The charm also hosts the `profile_management` package which aims to be abstracted to its own PyPi module in the
+future. The code for this should be able to be tested (unit/integration) in separate from the rest of the charm.
+
+For this reason we have developed the following poetry and tox environments:
+```bash
+# unit tests
+poetry install --with unit-library --no-root
+tox -e unit-library
+```
+
```

Comment:
I think this should be after the information about the charm in order to follow a general to specific logical order, since this is what the repository is about

Reply:
42fca23